scripts/schema/loader.py

```diff
@@ -251,3 +251,33 @@ def merge_fields(a, b):
             a[key].setdefault('fields', {})
             a[key]['fields'] = merge_fields(a[key]['fields'], b[key]['fields'])
     return a
+
+
+def load_yaml_file(file_name):
+    with open(file_name) as f:
+        return yaml.safe_load(f.read())
+
+
+# You know, for silent tests
+def warn(message):
+    print(message)
+
+
+def eval_globs(globs):
+    '''Accepts an array of glob patterns or file names, returns the array of actual files'''
```

Comment:
Nit: should we use `"` here instead of `'` to be consistent with the `exclude_filter`?

Reply:
Good catch @kgeller - I checked and that was also inconsistent elsewhere so I've normalized to `"""` as per [PEP-0257](https://www.python.org/dev/peps/pep-0257/)